src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -187,6 +226,18 @@ public JedisCluster(Set<HostAndPort> clusterNodes, JedisClientConfig clientConfi
     super(clusterNodes, clientConfig, maxAttempts);
   }
 
+  /**
+   * Creates a Jedis-Cluster with multiple Nodes/Instances. <br>
```

Comment:
Here, one of the multiples nodes/instances is used as an entry point (or to make the first contact) with the cluster.